core/src/main/java/nl/nn/adapterframework/jdbc/dbms/DbmsSupportFactory.java

```diff
@@ -77,13 +85,21 @@ public IDbmsSupport getDbmsSupport(Connection conn) {
 				return new MsSqlServerDbmsSupport();
 			}
 			if (Dbms.MYSQL.getProductName().equals(product)) {
+				if (productVersion.contains("MariaDB")) {
+					log.debug("Setting databasetype to MARIADB (using MySQL driver)");
+					return new MariaDbDbmsSupport();
+				}
 				log.debug("Setting databasetype to MYSQL");
 				return new MySqlDbmsSupport();
 			}
 			if (Dbms.MARIADB.getProductName().equals(product)) {
-				log.debug("Setting databasetype to MARIADB");
+				log.debug("Setting databasetype to MARIADB (using MariaDB driver)");
 				return new MariaDbDbmsSupport();
 			}
+			if (Dbms.POSTGRESQL.getProductName().equals(product)) {
+				log.debug("Setting databasetype to POSTGRESQL");
+				return new PostgresqlDbmsSupport();
+			}
```

Comment:
Kan dit niet mooier? we krijgen nu 6+ if statements... waarom dit zit niet in de enum?